src/transformers/commands/pt_to_tf.py

```diff
@@ -119,8 +165,22 @@ def run(self):
         repo = Repository(local_dir=self._local_dir, clone_from=self._model_name)
         repo.git_pull()  # in case the repo already exists locally, but with an older commit
 
+        # Load config and get the appropriate architecture -- the latter is needed to convert the head's weights
+        config = AutoConfig.from_pretrained(self._local_dir)
+        architectures = config.architectures
+        if architectures is None:  # No architecture defined -- use auto classes
+            pt_class = getattr(import_module("transformers"), "AutoModel")
+            tf_class = getattr(import_module("transformers"), "TFAutoModel")
+            self._logger.warn("No detected architecture, using auto classes")
+        else:  # Architecture defined -- use it
+            if len(architectures) > 1:
+                raise ValueError(f"More than one architecture was found, aborting. (architectures = {architectures})")
+            pt_class = getattr(import_module("transformers"), architectures[0])
+            tf_class = getattr(import_module("transformers"), "TF" + architectures[0])
```

Comment:
There should be some test here to raise a clear error message if the corresponding architecture in TF does not exist.